### ISSUE TITLE
Add missing REST options for second app server

### ISSUE THREAD

**Issue by brent-hartwig** [project-lux/lux-marklogic#212]
**Problem Description:**
Ever since the middle tier started splitting its backend requests between one of two backend app servers, one of the app servers had additional REST API configuration.  While we are not aware of any issues this has caused, we would like to match options.

**Expected Behavior/Solution:**
The tenant's modules database should receive two additional files --copies of `/Default/[tenant-name]-request-group-1/rest-api/options/lux-options.xml` and `/Default/[tenant-name]-request-group-1/rest-api/properties.xml` but with `[tenant-name]-request-group-2` in the URIs.

**Requirements:**
See above.

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

~- [ ] Wireframe/Mockup - Mike~
~- [ ] Committee discussions - Sarah~
~- [ ] Feasibility/Team discussion - Sarah~
- [x] Backend requirements - See above.
~- [ ] Frontend requirements- TBD~
~- [ ] Are new regression tests required for QA - Amy~
~- [ ] Questions~
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
Steps a developer may perform to verify:

1. Clear the tenant's modules database
2. Run the `mlDeployModules` task.
3. Verify `cts.uriMatch('\*request-group\*.xml')` returns four URIs.

**Dependencies/Blocks:**
This issue neither blocks nor is blocked by another issue.

**Related Github Issues:**
None

**Related links:**
None

**Wireframe/Mockup:**
N/A

**Comment by brent-hartwig** [comment]
Implemented in PR https://github.com/project-lux/lux-marklogic/pull/213

**Comment by brent-hartwig** [comment]
I learned yesterday the new `copyRestApiOptions` Gradle task may require an admin to run, despite being part of the non-admin portion of the deployment.  I added a try/catch to this task in the [updated-data-slices branch](https://github.com/project-lux/lux-marklogic/tree/updated-data-slices) where the message instructs to try again using an admin account.  It may save someone from having to investigate until we decide whether to address.  My best guess is a document permission issue.

cc: @clarkepeterf, @kamerynB, @xinjianguo
